### PR TITLE
Add sign color format to the configuration

### PR DESCRIPTION
**Pull request by montlikadani**
This does not know if it's right, but this may be useful because the server owner can edit the sign color to show it in color. And the rest of the sign jar needs to do this.

_(I could not add more commit because I'm still looking for how.)_

### CODE REVIEW

File: Essentials/src/com/earth2me/essentials/signs/SignEnchant.java [hunk 1]
```diff
@@ -24,24 +24,24 @@ protected boolean onSignCreate(final ISign sign, final User player, final String
         try {
             stack = sign.getLine(1).equals("*") || sign.getLine(1).equalsIgnoreCase("any") ? null : getItemStack(sign.getLine(1), 1, ess);
         } catch (SignException e) {
-            sign.setLine(1, "§c<item|any>");
+            sign.setLine(1, ChatColor.translateAlternateColorCodes('&', ess.getSettings().getErrorSignColor("<item|any>")));
```
Comment (LaxWasHere) [inline Essentials/src/com/earth2me/essentials/signs/SignEnchant.java:27]: No such method `ess.getSettings().getErrorSignColor()` ?

Comment (montlikadani) [reply Essentials/src/com/earth2me/essentials/signs/SignEnchant.java:27]: It's just an example of getting into the config.

**Comment by caojohnny** [comment]
I don't exactly agree with the intentions of this PR. I don't think anything useful would be added by being able to configure an error message that shouldn't be there, but that's just my opinion. I would say that if you're going to modify the color code(s), then go all the way and allow the message itself to be configured as well, but again, I'm not sure how this would be useful at all.

The format for the settings should not contain any parameters, you should be appending the color code in front of the message here.

**Comment by mdcfe** [comment]
Could you explain what this PR adds on its own? This evidently isn't finished, so we can't merge it even if there was a clear purpose to it.

**Comment by montlikadani** [comment]
@AgentTroll 
@md678685 
This feature is not so important. **It is only for the server owner to change the color of the sign in the configuration to see the color on the sign, for example. the [Spawn], [Warp].**

**Comment by caojohnny** [comment]
Yes, but you are only changing the color of the error message.

**Comment by montlikadani** [comment]
@AgentTroll 
I just want it.

**Comment by mdcfe** [comment]
Closing as there hasn't been any progress on this.

**Pull request closed without merging**